### DISCUSSION TITLE
Bad curved space example in documentation?

### DISCUSSION THREAD

**Discussion by ogfirstofhisname** [einsteinpy/einsteinpy#658] [General]
Documentation gives a bug walking along the surface of a cone, as an example of 2D curved space.
https://docs.einsteinpy.org/en/stable/metric.html#einstein-s-equation
The surface of a cone, however, is not a 2D curved space - it has a direct mapping (the Pac-Man) to a flat space and has zero curvature.

**Comment by JeS24** [comment]
@ogfirstofhisname Good catch. Do you want to open a small PR and fix the example - maybe use a spherical surface? The corresponding file is https://github.com/einsteinpy/einsteinpy/blob/main/docs/source/gr_primer.rst#notion-of-curved-space, with updates to it appearing at https://docs.einsteinpy.org/en/latest/gr_primer.html.
